app/models/calculator/average_searches_per_user.rb

```diff
@@ -0,0 +1,23 @@
+class Calculator::AverageSearchesPerUser
+  attr_reader :searches, :unique_pageviews
+
+  def self.calculate(*args)
+    new(*args).calculate
+  end
+
+  def initialize(searches:, unique_pageviews:)
+    @searches = searches
+    @unique_pageviews = unique_pageviews
+  end
+
+  def calculate
+    searches_float = searches.to_f
+    unique_pageviews_float = unique_pageviews.to_f
```

Comment:
Could we move this to the initialiser?

Reply:
I don't think so because it disrupts the return 0 if ...blank? checks. nil.to_f will always return 0.0 ruby-doc.org/core-2.5.0/NilClass.html#method-i-to_f